_posts/2021-12-17-glean-in-2021.md

```diff
@@ -0,0 +1,188 @@
+There's some work to be done next year to make local development easier again and to automate some of the tasks of upgading Glean across applications.
+
+[geckoview]: https://fnordig.de/2021/09/17/glean-geckoview/
+[crash]: https://fnordig.de/2021/11/01/crashes-and-a-buggy-glean/
+
+### Firefox on Glean (FOG)
+
+We invested a lot of time in making the Firefox integration better,
+fixing bugs and convincing people to migrate their instrumentation to the new Glean-based system.
+It's going slow, but we now have the [first few important metrics][fog-dictionary] in Glean pings coming from Firefox Desktop.
+Additionally Glean is the only telemetry solution in the Desktop Background Updater.
+Next year we hope to convince more people to start using Glean in Desktop,
+with the added benefit that this data will also be available in Firefox for Android.
+
+[fog-dictionary]: https://dictionary.telemetry.mozilla.org/apps/firefox_desktop
+[fog-bgupdate]: https://dictionary.telemetry.mozilla.org/apps/firefox_desktop_background_update
+
+### Glean.js
+
+A year ago [Glean.js was just started][gleanjs-initial].
+Thanks to Bea's tireless work it's now nearly feature-complete compared to the other Glean SDKs
+and used in production in applications & web extensions and soon even used to instrument websites.
+Next year Bea aims for a proper stable v1.0 release and hopefully more use across different products.
+It's a real alternative for those platforms where the inital set of SDKs don't cut it.
+
+Glean surely is becoming the one telemetry solution across Mozilla products.
+
+[gleanjs-initial]: https://github.com/mozilla/glean.js/commit/46f028fb4ea7b8f312daf4666904c81d0a3eb171
+
+### Tooling: GLAM, Dictionary, Looker
+
+These are the tools I'm much less directly involved in,
+but they make up an important part of the Glean ecosystem.
+Afterall these are where users actually get to analyse at the data they collect.
+
+GLAM currently only serves data for Firefox Desktop using legacy telemetry data as well as Firefox for Android using Glean data.
+Things are in place to enable more Glean-powered products soon, once we [iron out some details around data expectations from applications][build-date].
+
+The [Glean Dictionary][dictionary] saw near-weekly releases
+and went from "promising prototype" into "valuable production tool" this year,
+thanks to a lot of work from [wlach] and [linh].
+It was the basis of several initiatives such as [metric annotations][annotations] and [tagging support][tags].
+These things will be helpful to make data at Mozilla more discoverable and better documented.
+
+The primary tool for analyzing data at Mozilla is now [Looker].
+Folks have been very busy making more and more data accessible to both data experts and non-experts alike[^1].
+Glean data is (more or less) automatically available in Looker explores.
+Instead of bring up up adhoc dashboards for initiatives when needed we can recommend to build it all on Looker.
```

Comment:
Hmm, I'd suggest rewording this slightly as it reads a little strangely. Here's my attempt:

```suggestion
Instead of bringing up adhoc dashboards for initiatives when needed we can usually recommend that people just build things in Looker.
```